src/main/webapp/add_event_page/add_event.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Add Event</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDlz0NQ1mc6_UU6OHExGKIrpnkJ5q9BqPs"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload="initMap()">
+    <div id="header">
+      <h1>Add New Event</h1>
+    </div>
+
+    <div id="form">
+        <form action="/new_event.php">
+            <label for="name">Name:</label>
+            <input type="text" id="name" name="name"><br>
+            <label for="desc">Description:</label>
+            <input type="text" id="desc" name="desc"><br>
+            <label for="loc">Building Name:</label>
+            <input type="text" id="loc" name="loc"><br>
+            <label for="date">Date:</label>
+            <input type="text" id="date" name="date"><br>
+            <label>Event Type:</label>
+            <select name="ev_type" id="ev_type">
```

Comment:
Would it be easier to have the subject and the event type be separate properties? I think this is fine, but I'm thinking about it because it's different from how I set up the filter options in the directory page.

Reply:
We can keep this for now and discuss in more depth during our next meeting to come to an agreement on something that's consistent. This is all good practice on coming to an agreement across different components on a website!